1.22-minimal/Dockerfile.c9s

```diff
@@ -0,0 +1,98 @@
+FROM quay.io/centos/centos:stream9 AS build
+
+RUN mkdir -p /mnt/rootfs
+ADD https://copr.fedorainfracloud.org/coprs/hhorak/nginx-micro/repo/centos-stream-9/hhorak-nginx-micro-centos-stream-9.repo /etc/yum.repos.d/hhorak-nginx-micro-centos-stream-9.repo
+RUN MICRO_PKGS="coreutils-single glibc-minimal-langpack" && \
+    INSTALL_PKGS="$MICRO_PKGS nginx-core findutils hostname nss_wrapper-libs envsubst bind-utils" && \
+    dnf install --installroot /mnt/rootfs $INSTALL_PKGS --releasever 9 --setopt install_weak_deps=false --nodocs -y && \
+    dnf -y --installroot /mnt/rootfs clean all && \
+    rm -rf /mnt/rootfs/var/cache/* /mnt/rootfs/var/log/dnf* /mnt/rootfs/var/log/yum.*
+
+FROM quay.io/centos/centos:stream9
```

Comment:
Would use `FROM scratch` here, as there is no reason to layer it on top of a full sized image. We already have all we need from the previous build phase.

Reply:
will try, thanks for suggestion